### ISSUE TITLE
Network Error, retrying...

### ISSUE THREAD

**Issue by dagsix** [tgwizard/sls#214]
Hello,
I read this is no longer active but hoping you might still be able to answer a question. I'm using SLS on a old HTC Desire and when I try to sign into last.fm I get "Network Error, retrying...". 

Trying to re-purpose my old phone as a music player and would love it to have last.fm functionality. 

Any thoughts?  
Thanks!

**Comment by HumbleBeeBumbleBee** [comment]
http://status.last.fm/

**Comment by HumbleBeeBumbleBee** [comment]
This is not our fault. It's making me angry too!

**Comment by HumbleBeeBumbleBee** [comment]
I will leave this post open until Last.fm fixes their website issues, and users report SLS as working with last.fm again.

I know it is not SLS because Libre.fm is scrobbling perfectly, also the official last.fm application for Windows is buggy but still scrobbling some how.

**Comment by HumbleBeeBumbleBee** [comment]
Two users reporting the same issue on Google Play.

**Comment by inverse** [comment]
Damn - they forced their new website upon us :(

**Comment by dagsix** [comment]
Oh! Sorry didn't even think it could be something on last.fm's end. Cheers. I'll wait it out. Apps looks great.

**Comment by HumbleBeeBumbleBee** [comment]
http://status.last.fm/
Last Updated: Fri, 21 Aug 2015 03:09:07 +0200
Submissions == ok
YAY!
They were working earlier for me, but now it's official.

**Comment by riversnake** [comment]
Unfortunately, the error remains on my device.  It worked perfectly before and the Libre.fm sign in / scrobbling remains functional.  The internet connection (only Wi-Fi available) is not indicating any problems.

I attempted to clear the app data, uninstalled and reinstalled from Google Play, and also installed v1.4.9 just in case.  In all instances the error continues with Last.fm (but not Libre.fm).

The device in question is a Samsung SM-T700 with Android 5.0.2.  It works fine on my phone (HTC One M8 with Android 5.0.1).

Many thanks.

**Comment by HumbleBeeBumbleBee** [comment]
Have you tested different music players?
Could you share what music app(s) and version(s) you're using, and your results?

**Comment by riversnake** [comment]
I haven't tried different music players, but did so on your suggestion.

My primary music player is GoneMAD Music Player (2.0.7)
I also cycled through the following:
ES FIle Explorer's built in music player (3.2.5.5)
Google Play Music (6.0.1951S)
Stock music player (unsure about name / version)
VLC (1.3.2)

Not sure why, but SLS signed in and is seemingly scrobbling as it should from all the above.  I did not keep track during which playback SLS signed in though.  Also, during one of the playbacks, I noted my device's time was incorrect by about 2 minutes (ahead) which I decided to fix.  Perhaps this also contributed to the initial issue?

Nevertheless, thanks for the assistance and the great app.
